app/javascript/react/src/components/BookTile.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
+import { Link } from 'react-router';
 
 const BookTile = (props) => {
```

Comment:
We are using BookTile on our Book Index page, which is different than our home page (BookHomeTile) in that it just lists the books by title (eventually alphabetically by title). BookHomeTile has more information passed down to it in order to make the home page more "rich".